src/it/resources/application-it.conf

```diff
@@ -0,0 +1,62 @@
+include "persistence-jdbc"
+
+akka {
+
+  stdout-loglevel = "INFO"
+  loglevel = "INFO"
+  use-slf4j = on
+
+  actor.provider = cluster
+
+  remote.classic.netty.tcp.port = 0
+
+  remote.artery {
+    canonical.port = 0
+    canonical.hostname = 127.0.0.1
+  }
+
+  coordinated-shutdown {
+    terminate-actor-system = off
+    run-by-actor-system-terminate = off
+    run-by-jvm-shutdown-hook = off
+  }
+
+  cluster {
+    jmx.multi-mbeans-in-same-jvm = on
+    run-coordinated-shutdown-when-down = off
+    sharding {
+      number-of-shards = 5
+    }
+  }
+
+  typed {
+    stash-capacity = 200000
+  }
+}
+
+key-management {
```

Comment:
We used to name this key with the service name.
Here and in other configs, we still refer to the old service name.
The key should be changed in `authorization-management`.
Anyway, it's not urgent.

Reply:
updated 👍